Wikipedia/Categories/NSAttributedString+WMFModifyParagraphs.m

```diff
@@ -0,0 +1,25 @@
+//  Created by Monte Hurd on 8/17/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "NSAttributedString+WMFModifyParagraphs.h"
+
+@implementation NSAttributedString (WMFModifyParagraphs)
+
+- (NSAttributedString*)wmf_attributedStringWithParagraphStylesAdjustments:(void (^)(NSMutableParagraphStyle* existingParagraphStyle))block {
+    if (!block) {
+        return self;
+    } else {
```

Comment:
style nit-pick: if you return early from a method, omit the `else` clause to remove unnecessary indentation